src/hooks/sdk/useMediaStreamMetrics.tsx

```diff
@@ -6,15 +6,17 @@ import { useEffect, useState } from 'react';
 
 import { useAudioVideo } from '../../providers/AudioVideoProvider';
 
-function isValidMetric(metric: number): boolean {
-  return !Number.isNaN(metric);
-}
-
 interface MediaStreamMetrics {
   audioPacketsSentFractionLossPercent: number | null; // Percentage of audio packets lost (1s) from client to server
   audioPacketsReceivedFractionLossPercent: number | null; // Percentage of audio packets lost from server to client
+  audioSpeakerDelayMs: number | null;
+  audioUpstreamRoundTripTimeMs: number | null;
+  audioUpstreamJitterMs: number | null;
+  audioDownstreamJitterMs: number | null;
+  currentRoundTripTimeMs: number | null;
```

Comment:
We are not able to fetch these metrics from JS SDK at this moment and their values are 0. 
Once JS SDK is released and we upgrade the JS SDK version in React SDK, they will be fetched successfully. Tested by install the JS SDK tarball.

Reply:
Can resolve as I see you updated JS SDK to 3.1.0 :)